mobile/src/bundles/talent/components/profile-screen.profile-form/profile-screen-profile-form.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+
+import { logout } from '~/bundles/auth/store/actions';
+import {
+    AutocompleteSelector,
+    Button,
+    CheckboxGroup,
+    FormField,
+    Input,
+    Pressable,
+    ScrollView,
+    Selector,
+    Slider,
+    View,
+} from '~/bundles/common/components/components';
+import { ButtonType } from '~/bundles/common/enums/enums';
+import {
+    useAppDispatch,
+    useAppForm,
+    useState,
+} from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+import { EXPERIENCE_YEARS } from '~/bundles/talent/components/profile-form/constants/constants';
+import { ProfileStepValidationRule } from '~/bundles/talent/enums/enums';
+import { type ProfileStepDto } from '~/bundles/talent/types/types';
+import { ProfileStepValidationSchema } from '~/bundles/talent/validation-schemas/validation-schemas';
+
+import {
+    EMPLOYMENT_TYPE_OPTIONS,
+    JOB_TITLE_OPTIONS,
+    LOCATION_OPTIONS,
+} from './constants/constants';
+import { styles } from './styles';
+
+type Properties = {
+    onSubmit: (payload: ProfileStepDto) => void;
+    userData: ProfileStepDto;
+};
+
+const ProfileScreenProfileForm: React.FC<Properties> = ({
```

Comment:
Can we reuse forms from onboarding?

Reply:
They have the same logic as step 1-4, on the each tab screen you just pass initial values to the forms